docs/faq_debug.md

```diff
@@ -110,6 +110,19 @@ If you can't get this 'Listening:' message try building with `CONSOLE_ENABLE=yes
 You may need privilege to access the device on OS like Linux.
 - try `sudo hid_listen`
 
+On many Linux distros you can avoid having to run hid_listen as root
+by creating a file called `/etc/udev/rules.d/70-hid-listen.rules` with
+the following content:
+
+```
+SUBSYSTEM=="hidraw", ATTRS{idVendor}=="abcd", ATTRS{idProduct}=="def1", TAG+="uaccess", RUN{builtin}+="uaccess"
```

Comment:
Would it be possible to filter based on HID usage page & usage instead? That would allow you to add this rule once and have it work for any Console-enabled board.

Reply:
I don't see any docs referring to being able to use usage page and ID. Perhaps we get the docs in now and amend later?

Reply:
It's not possible to use any inbuilt attributes to match against usage page and id. I've seen examples of people adding an executable written in C which does the matching. The result is then returned to udev where it can be filtered using rules as normal. It feels out of scope for a manual though.

Are there any security implications of adding uaccess to the device like this?

Reply:
Definitely feels out of scope.
I see no reason to hold back this merge, given that none of it is automated and requires manual intervention from the user in order to get things working. At this stage I'm happy with it as-is, but will defer to @fauxpark if they want to make further comment.

Reply:
Security concerns, probably not, I just think it's a bit of a hassle to have to add a rule for each device. Maybe a script to automatically generate the file from a user-supplied list and reboot udev? But that would be out of scope here too.

If there's no other way to be more precise, then 👍